trade-finance-manager-ui/styles/table.scss

```diff
@@ -5,3 +5,60 @@
 .govuk-table__cell--border-top {
   border-top: solid 1px govuk-colour('mid-grey');
 }
+
+.gov-table__grey-table-column {
+  background: #F3F2F1;
+}
+
+.gov-table_cell-vertical-align-middle {
+  text-align: center;
+  vertical-align: middle;
+}
+
+.gov-table_cell--border-bottom-transparent {
+  border-bottom: 1px solid transparent;
+}
+
+.facilities-table td {
+  text-align: right !important;
```

Comment:
Do we need `!important`, can we use `id` or ` class` instead please?

Reply:
it was like this before - I've just removed the other files and moved the code here. I'm not sure why it needs `!important` either

Reply:
I found out why - the table is generated using the govuk component. so adding an `id` or `class` to a `td` element is not possible